source/client-side-encryption/client-side-encryption.rst

```diff
@@ -336,14 +336,17 @@ MongoClient Changes
       private Optional<MongoClient> internal_client; // An internal MongoClient. Created if no external keyVaultClient was set, or if a metadataClient is needed
    }
 
+.. _AutoEncryptionOpts:
+.. code-block:: typescript
+
    class AutoEncryptionOpts {
       keyVaultClient: Optional<MongoClient>;
       keyVaultNamespace: String;
-      kmsProviders: Map<String, Map<String, Value>>;
+      kmsProviders: KMSProvidersMap;
```

Comment:
```suggestion
      kmsProviders: KMSProviders;
```

Since `KMSProvidersMap` and `KMSProviderTLSOptions` represent a map from KMS provider name, suggest making the names consistent: `KMSProviders` and `KMSProvidersTLSOptions`?